distribution/resources/bin/micro-gw.bat

```diff
@@ -52,37 +50,20 @@ SET BALLERINA_HOME=%MICROGW_HOME%\lib\platform
 if NOT EXIST %BALLERINA_HOME% SET BALLERINA_HOME="%MICROGW_HOME%\lib"
 
 SET PATH=%BALLERINA_HOME%\bin\;%PATH%
-if %verbose%==T ECHO BALLERINA_HOME environment variable is set to %BALLERINA_HOME%
-if %verbose%==T ECHO MICROGW_HOME environment variable is set to %MICROGW_HOME%
 
 REM Check JAVA availability
-:checkJavaHome
-	if "%JAVA_HOME%" == "" goto noJavaHome
-	if NOT EXIST "%JAVA_HOME%\bin\java.exe" goto noJavaHome
-goto checkJava
+if EXIST "%JAVA_HOME%" (
+	ECHO JAVA_HOME: %JAVA_HOME%
+) else (
+	SET JAVA_HOME=%MICROGW_HOME%\lib\jdk8u202-b08-jre
+)
```

Comment:
IMO, we should check if an executable java file is inside the JAVA_HOME as well. And besides that, we should check if it is java 1.8